### DISCUSSION TITLE
Pull dropdown results from another table

### DISCUSSION THREAD

**Discussion by hydroxy** [Budibase/budibase#8112] [Help] [answered]
![image](https://user-images.githubusercontent.com/6318064/193720167-f923dc4a-5304-40d6-aab2-9a09138913f5.png)
How can I create a form that has a dropdown that pulls its results from another table. In the image above the processType field should display names such as 'Health Assessment' and 'Diary Entry', however instead it uses the row _id numbers in the textbox.

In the above image I am creating a new 'process' table entry it should pull the dropdown information from the 'processType' table. I have the relationship set up in mySQL Budibase section, however it does not give me the option to display a name field and instead just shows the incomprehensible ID field.

**Comment by andz-bb** [comment, marked as answer]
Hey @hydroxy, I spun up a mySQL db there to test it out and I was getting the same. This issue #7867 looks to be the problem. The fix for it #7868 has just been merged into master and deployed to prod. I have tested it out on my end (version 2.0.18) and it looks good! After updating make sure the column you would like to display in the picker has _Use as table display column_ enabled. Thanks @melohagan for the fix!

**Reply by melohagan** [reply]
I think the break was my fault to be honest, but good to be fixed! 😅

**Reply by hydroxy** [reply]
Thanks for the help. Issue is resolved for me 👍
